### DISCUSSION TITLE
Is it possible to load obj file in SceneformMaintained ?

### DISCUSSION THREAD

**Discussion by monsterbrain** [sceneview/sceneform-android#210] [Q&A] [answered]
I would like to load an OBJ file in sceneform. Partly because, this obj file is created dynamically (text related).

I've seen that it might be possible with sfa/sfb file with model defined, but the support has been dropped as I read in migration notes.

The **Mesh** data is obtained dynamically, so if it's possible to directly load Mesh data to a modelRenderable.

**Comment by RGregat** [comment, marked as answer]
Sceneform has no OBJ-Parser/Loader, but you could take a look into [ShapeFactory](https://github.com/ThomasGorisse/SceneformMaintained/blob/master/core/src/main/java/com/google/ar/sceneform/rendering/ShapeFactory.java) how to build your own Renderable with just vertices, uvs and indices. I would say the **makeCube** function is the easiest one to understand the process.

**Comment by monsterbrain** [comment]
As a workaround, I used obj-to-gltf java library for generating gltf file from obj and successfully loaded in sceneform.
